SmartEVSE-3/src/evse.cpp

```diff
@@ -5031,22 +5238,40 @@ void setup() {
 
     CP_ON;           // CP signal ACTIVE
 
-
+    //mongoose
+    mg_mgr_init(&mgr);  // Initialise event manager
 }
 
 void loop() {
-    //this loop is for non-time critical stuff that needs to run approx 1 / second
-    vTaskDelay(1000 / portTICK_PERIOD_MS);
-    getLocalTime(&timeinfo, 1000U);
-    if (!LocalTimeSet && WIFImode == 1) {
-        _LOG_A("Time not synced with NTP yet.\n");
+
+    static unsigned long lastNtpCheck = 0;
+    if (millis() - lastNtpCheck >= 1000) {
+        lastNtpCheck = millis();
+        //this block is for non-time critical stuff that needs to run approx 1 / second
+        getLocalTime(&timeinfo, 1000U);
+        if (!LocalTimeSet && WIFImode == 1) {
+            _LOG_A("Time not synced with NTP yet.\n");
+        }
     }
 
     if (shouldReboot) {
         delay(1000);
         ESP.restart();
     }
 
+    mg_mgr_poll(&mgr, 1000);
```

Comment:
How about I move the mgr_init, mgr_poll and mgr_free to a separate commit in my master, and insulate it from OCPP ?
mgr_init should be in the wifi event "connected" since we need it to stop/start when the captive portal is started....